tests/e2e/scripts/pool1A.json

```diff
@@ -2,6 +2,5 @@
     "weights": "5ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518,5uosmo",
     "initial-deposit": "499404ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518,500000uosmo",
     "swap-fee": "0.01",
-    "exit-fee": "0.01",
```

Comment:
e2e is failing because this is removed.

Basically, it tries to create a pool for pre-upgrade Osmosis (v14) that still has and expects this parameter.

I suggest copying this file and naming it `pool1A-legacy.json` and using it in: https://github.com/osmosis-labs/osmosis/blob/ae97cec94f665b44365e909e9031edc36d1d4f34/tests/e2e/configurer/upgrade.go#L128

Also, please do [the same](https://github.com/osmosis-labs/osmosis/blob/ae97cec94f665b44365e909e9031edc36d1d4f34/tests/e2e/configurer/upgrade.go#L130) for `pool1B.json`. Please create an issue to track the removal of these legacy files later